src/features/NavBar/Navigation.jsx

```diff
@@ -1,8 +1,8 @@
 import { CiLocationOn, CiMap } from 'react-icons/ci';
 import { useContext } from 'react';
-import { AppContext } from '../../store/AppContext';
+import { AppContext } from '../../index';
 
-const Navigation = () => {
+export const Navigation = () => {
```

Comment:
this is also part of the interface

pls don't make it like onion. 

features/CitySec 
features/Forecast
features/Info
features/Navigation
features/Search
etc.